specs/core/0_beacon-chain.md

```diff
@@ -1689,7 +1691,6 @@ def process_deposit(state: BeaconState, deposit: Deposit) -> None:
             activation_epoch=FAR_FUTURE_EPOCH,
             exit_epoch=FAR_FUTURE_EPOCH,
             withdrawable_epoch=FAR_FUTURE_EPOCH,
-            effective_balance=min(amount - amount % EFFECTIVE_BALANCE_INCREMENT, MAX_EFFECTIVE_BALANCE)
```

Comment:
Should leave this line imo.
When a validator as an initial deposit, it makes sense to set their effective immediately.
Also keeps the tests passing.

Reply:
I think it's a nice property to know stable low-resolution balances (i.e. only changing stepwise once per epoch at most) for non-activated validators. It helps people monitoring for other reasons. Besides, de-activated validators get a effective balance too, and that is updated as well. It's much more consistent to keep it as-is.

Reply:
Added back (with the missing comma added). Not fully satisfied but can't think of a better solution right now.